.ado/android-pr.yml

```diff
@@ -23,137 +23,67 @@ jobs:
         submodules: recursive # set to 'true' for a single level of submodules or 'recursive' to get submodules of submodules
         persistCredentials: false # set to 'true' to leave the OAuth token in the Git config after the initial fetch
 
+      - task: UseNode@1
+        inputs:
+          version: '10.x'
```

Comment:
Does it work with 12.x?  10.x is really old at this point.

Reply:
I don't see any reason for the task to not work with 12.x .. I'll push an update now.